node-notes-cli/app.js

```diff
@@ -0,0 +1,65 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const [, , command, notesOrId, edit] = process.argv;
```

Comment:
Cute names. I had trouble naming these variables, too, since they have different meaning for different commands, so I gave up and just called them `args`!

In a real, sophisticated CLI you would do all the command line argument parsing using an NPM package, like `commander`.